f2/src/forms/HowDidItStartForm.js

```diff
@@ -127,10 +183,53 @@ export const HowDidItStartForm = props => {
               </Stack>
             </Control>
 
+            <Control as="fieldset" name="whenDidItStart">
+              <FormLabel as="legend" htmlFor="whenDidItStart" mb={2}>
+                <Trans id="whenDidItStart.label" />
+              </FormLabel>
+              <Stack spacing={4} shouldWrapChildren>
+                {whenDidItStart.map(key => {
+                  return (
+                    <Box key={key}>
+                      <RadioButtonArrayControl
+                        name="whenDidItStart"
+                        value={key}
+                        isChecked={timeline.whenDidItStart.includes(key)}
+                      >
+                        {i18n._(key)}
+                      </RadioButtonArrayControl>
+                    </Box>
+                  )
+                })}
+              </Stack>
+            </Control>
+
+            <Control as="fieldset" name="howManyTimes">
+              <FormLabel as="legend" htmlFor="howManyTimes" mb={2}>
+                <Trans id="howManyTimes.label" />
+              </FormLabel>
+              <Stack spacing={4} shouldWrapChildren>
+                {howManyTimes.map(key => {
+                  return (
+                    <Box key={key}>
+                      <RadioButtonArrayControl
+                        name="howManyTimes"
+                        value={key}
+                        isChecked={recurrenceCheck.howManyTimes.includes(key)}
+                      >
+                        {i18n._(key)}
+                      </RadioButtonArrayControl>
+                    </Box>
+                  )
+                })}
+              </Stack>
+            </Control>
+            <P>
+              <Trans id="whatWasAffectedForm.nextPage" />
+            </P>
+
             <NextAndCancelButtons>
-              <Trans id="howDidItStartPage.nextButton">
-                Next: ??
-              </Trans>
+              <Trans id="whatWasAffectedForm.nextButton" />
```

Comment:
Should this be `howDidItStartPage.nextButton`?

Reply:
Changes are now embedded into the last commit + page links to the proper workflow